frontend/src/app/university/feature/page-contact-form/page-contact-form.component.ts

```diff
@@ -47,24 +47,27 @@ import { first, switchMap } from 'rxjs';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class PageContactFormComponent {
-  private readonly _service = inject(PageService);
+  private readonly _service = inject(TicketService);
   private readonly _pageId$ = resourceIdFromRoute('pageId');
 
   readonly form = inject(FormBuilder).nonNullable.group({
-    pageId: [],
-    name: ['', [Validators.required, Validators.maxLength(255)]],
-    subject: ['', [Validators.required, Validators.maxLength(255)]],
-    email: [
+    title: ['', [Validators.required, Validators.maxLength(255)]],
+    requesterEmail: [
       '',
       [Validators.required, Validators.maxLength(255), Validators.email],
     ],
-    content: ['', [Validators.required, Validators.maxLength(255)]],
+    description: ['', [Validators.required, Validators.maxLength(2000)]],
```

Comment:
> Bumped question length to 2000 as 255 seemed very little, I wonder if we need this sort of limit in the first place.

I set this to 255 as the `Ticket.java` has a `String description` field, which default limit is 255 characters. `Response` content's also `String`. If you want to extend this limitation, I think you must adjust the model's properties to Postgres text column type.

Reply:
Sure I'll revert this change at least for the time being and address this sometime else